packages/kit/src/runtime/client/remote-functions/form.svelte.js

```diff
@@ -429,13 +434,18 @@ export function form(id) {
 			},
 			validate: {
 				/** @type {RemoteForm<any, any>['validate']} */
-				value: async ({ includeUntouched = false } = {}) => {
+				value: async ({ includeUntouched = false, submitter } = {}) => {
 					if (!element) return;
 
 					const id = ++validate_id;
 
 					const form_data = new FormData(element);
 
+					const submitter_name = submitter?.getAttribute('name');
+					if (submitter_name) {
+						form_data.append(submitter_name, submitter?.getAttribute('value') ?? '');
+					}
+
```

Comment:
See https://github.com/sveltejs/kit/pull/14475/files#r2373940460
```suggestion
const form_data = new FormData(element, submitter);
```